owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -738,9 +728,12 @@ private void listDirectoryWithAnimationDown(final OCFile file) {
     }
 
     private boolean isInPowerSaveMode() {
-        PowerManager powerManager = (PowerManager)
-                getActivity().getSystemService(Context.POWER_SERVICE);
-        return  powerManager.isPowerSaveMode();
+        PowerManager powerManager = (PowerManager) requireActivity().getSystemService(Context.POWER_SERVICE);
+        if (powerManager != null) {
+            return powerManager.isPowerSaveMode();
+        } else {
+            return false;
+        }
```

Comment:
It is and I changed it.

Btw,  I would rater convert every touched file to Kotlin. But with this https://github.com/owncloud/android/pull/2680 experience, it's a task of someone else

Reply:
Thanks for the change. About converting files to kotlin, i consider that it could be great but only if an optimization is performed. I see no point in migrating to kotlin and keep java style.